Libraries/Utilities/NativeAppearance.js

```diff
@@ -26,6 +26,7 @@ export interface Spec extends TurboModule {
   // types.
   /* 'light' | 'dark' */
   +getColorScheme: () => ?string;
+  +setColorScheme: (colorScheme: ?string) => void;
```

Comment:
In the current scheme, where the color is always "unspecified" instead of null, we would not want to mark this as nulable.